<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="networkMissionTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1211</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="MobileNodesTab">
     <attribute name="title">
      <string>Mobile Nodes</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1181</width>
        <height>631</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCommandLinkButton" name="addNewMobileNode">
         <property name="text">
          <string>Add Mobile Node</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="selectMobileNodeCombo">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="mobileNodeNameLabel">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <italic>false</italic>
           <strikeout>false</strikeout>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="tempSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="StaticNodesTab">
     <attribute name="title">
      <string>Static Nodes</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="missionTab">
     <attribute name="title">
      <string>Mission</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1121</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="container" columnstretch="500,500">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="columnCount">
          <number>2</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="mapWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <zorder>gridLayoutWidget_3</zorder>
     <zorder>mapWidget</zorder>
     <zorder>mapWidget</zorder>
     <zorder>mapWidget</zorder>
     <zorder>mapWidget</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGroundStation">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAct"/>
   </widget>
   <addaction name="menuGroundStation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAct">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
